### ISSUE TITLE
Error code importing cifar10 (Python)

### ISSUE THREAD

**Issue by HenriqueOliveira18** [HenriqueOliveira18/PrimeLayerInternship#1]
I'm currently learning about Image Recognition and Classification in Python with Tensorflow and Keras for my internship on Prime Layer. 
I've study and copied the code from an article ( [https://stackabuse.com/image-recognition-in-python-with-tensorflow-and-keras/](url) ) but for some reason i can't run it because it gives me errors. I suspect that the main error is with the import of the dataset _cifar10_. Can someone help me?

The code i'm running is:

      import numpy
      from tensorflow import keras
      from keras.constraints import maxnorm
      from keras.utils import np_utils #Contains the function to_categorial().
      
      seed  = 21
      
      from keras.datasets import cifar10
      
      # Loading in the data (integers).
      (x_train, y_train), (x_test, y_test) = cifar10.load_data()
      
      print(x_train.shape)
      # Normalize the inputs from 0-255 to between 0 and 1 by dividing.
      X_train = x_train.astype('float32')
      X_test = x_test.astype('float32')
      X_train = X_train/255
      X_test = X_test/255
      
      # One-hot encode outputs.
      y_train = np_utils.to_categorial(y_train)
      y_test = np_utils.to_categorical(y_test)
      class_num = y_test.shape[1]
      
      # Creating the Sequential model and adding layers.
      # The first layers takes in the inputs and run convolutional filters on them.
      model = keras.Sequential([
          # First block
          keras.layers.Conv2D(32, (3,3), input_shape=X_train.shape[1:], padding='same', activation='relu'),    # padding='same' means we aren't changing the size of the image.
          keras.layers.Dropout(0.2),   # Dropout layer to prevent overfitting (drops 20%).
          keras.layers.BatchNormalization(),  # Normalizes the inputs heading into the next layer, ensuring that the network always creates activations with the same distribution that we desire.
      
          # Second block
          keras.layers.Conv2D(64, 3, padding='same', activation='relu'),   # Larger filter, which allows it to find patterns in greater detail and abstract further.
          keras.layers.MaxPooling2D(2),
          keras.layers.Dropout(0.2),
          keras.layers.BatchNormalization(),
      
          # Third block
          keras.layers.Conv2D(64, 3, padding='same', activation='relu'),   # Larger filter, which allows it to find patterns in greater detail and abstract further.
          keras.layers.MaxPooling2D(2),
          keras.layers.Dropout(0.2),
          keras.layers.BatchNormalization(),
      
          # Fourth block
          keras.layers.COnv2D(128, 3, activation='relu', padding='same'),
          keras.layers.Dropout(0.2),
          keras.layers.BatchNormalization(),
      
          # Flatting the data
          keras.layers.Flatten(),
          keras.layers.Dropout(0.2),
      
          # Densely connected layers
          keras.layers.Dense(32, activation='relu'),
          keras.layers.Dropout(0.3),
          keras.layers.BatchNormalization(),
          keras.layers.Dense(class_num, activation='softmax') # Softmax selects the neuron with the highest probability as its outputs, voting that the image belongs to that class.
          ])
      
      model.compile(loss='categorical_crossentropy', optimizer='adam', metrics=['accuracy','val_accuracy'])   # Keep tracking of the metrics Accuracy and Validation Accuracy.
      
      print(model.summary)

My python version is: 3.11.1 64-bit
My Tensorflow version is: 2.12.0rc1
